apps/aehttpclient/lib/client.ex

```diff
@@ -90,14 +89,14 @@ defmodule Aehttpclient.Client do
     get(uri <> "/tx_pool/#{acc}", :acc_txs)
   end
 
-  @spec handleResponse(:block, map(), map()) :: {:ok, map()}
-  defp handleResponse(:block, body, _headers) do
+  @spec handle_response(:block, map(), map()) :: {:ok, map()}
```

Comment:
`@spec handle_response(req_kind(), map(), list(map())) :: {:ok, map()}`
This spec can be defined once at the top, and then doesn't need to be repeated for each atom defined in `req_kind`